airflow/models.py

```diff
@@ -1238,87 +1113,87 @@ def run(
                 session.merge(self)
                 session.commit()
                 logging.info("Queuing into pool {}".format(self.pool))
-                return
+            return
 
-            # print status message
-            logging.info(HR + msg + HR)
-            self.try_number += 1
+        # print status message
+        logging.info(HR + msg + HR)
+        self.try_number += 1
 
-            if not test_mode:
-                session.add(Log(State.RUNNING, self))
-            self.state = State.RUNNING
-            self.end_date = None
-            if not test_mode:
-                session.merge(self)
-            session.commit()
+        if not test_mode:
+            session.add(Log(State.RUNNING, self))
+        self.state = State.RUNNING
+        self.end_date = None
+        if not test_mode:
+            session.merge(self)
+        session.commit()
 
-            # Closing all pooled connections to prevent
-            # "max number of connections reached"
-            settings.engine.dispose()
-            if verbose:
-                if mark_success:
-                    msg = "Marking success for "
-                else:
-                    msg = "Executing "
-                msg += "{self.task} on {self.execution_date}"
+        # Closing all pooled connections to prevent
+        # "max number of connections reached"
+        settings.engine.dispose()
+        if verbose:
+            if mark_success:
+                msg = "Marking success for "
+            else:
+                msg = "Executing "
+            msg += "{self.task} on {self.execution_date}"
 
-            context = {}
-            try:
-                logging.info(msg.format(self=self))
-                if not mark_success:
-                    context = self.get_template_context()
-
-                    task_copy = copy.copy(task)
-                    self.task = task_copy
-
-                    def signal_handler(signum, frame):
-                        '''Setting kill signal handler'''
-                        logging.error("Killing subprocess")
-                        task_copy.on_kill()
-                        raise AirflowException("Task received SIGTERM signal")
-                    signal.signal(signal.SIGTERM, signal_handler)
-
-                    self.render_templates()
-                    task_copy.pre_execute(context=context)
-
-                    # If a timout is specified for the task, make it fail
-                    # if it goes beyond
-                    result = None
-                    if task_copy.execution_timeout:
-                        with timeout(int(
-                                task_copy.execution_timeout.total_seconds())):
-                            result = task_copy.execute(context=context)
-
-                    else:
+        context = {}
+        try:
+            logging.info(msg.format(self=self))
```

Comment:
man. github needs to get better at diffin' tabs. It's good to get 4 spaces back in this heavy logic area. We need a TODO to refactor this method into smaller chunks but that's out of scope for this PR